scripts/tools.mk

```diff
@@ -8,11 +8,13 @@ SILENT?=@
 THISDIR:=$(dir $(lastword $(MAKEFILE_LIST)))
 
 UNAME:=$(shell uname)
-ifeq ($(UNAME),$(filter $(UNAME),Linux Darwin))
+ifeq ($(UNAME),$(filter Linux Darwin MINGW%,$(UNAME)))
```

Comment:
I assume this trailing % is intentional?

Reply:
It is, msys2 uname reports either 'MINGW64_NT-10.0-19042' or 'MINGW32_NT-10.0-19042' in which the last part of it changes with the windows os version.

Reply:
Ah, didn't know it's wildcard like that.